src/components/safe-apps/SafeAppList.tsx

```diff
@@ -28,6 +36,24 @@ const SafeAppList = () => {
 
   const debouncedSearchQuery = useDebounce(searchQuery, 500)
 
+  const wcLink: LinkProps['href'] = useMemo(() => {
+    const WC_APP_ID = 11
```

Comment:
First of all, the id is different on staging and prod. Secondly, hardcoding it here is no bueno.

Reply:
I would add a new tag, e.g. `search-fallback` or `walletconnect` and use that.
Let's not add for now, and leave the search how it was.